server/deployment/build.sh

```diff
@@ -0,0 +1,11 @@
+
+source ./environment.sh
+
+cd ../..
+echo "Building Arcs Master Docker Image..."
+docker build . -t $BUILD_LABEL
```

Comment:
Check return codes here and below? Similar rest of scripts (could do in follow-on PR)